docs/requirements.txt

```diff
@@ -1,9 +1,12 @@
 # pin docutils at 0.15.2 to avoid transitive dependency conflict with botocore (requires < 0.16)
 docutils==0.15.2
-sphinx==2.4.4
-sphinx-argparse>=0.2.5
-git+https://github.com/determined-ai/determined_sphinx_theme.git@9edfa7c6ce6926def9fc69b8ddd7666f3419a907
-sphinx-sitemap>=2.2.0
+
 pillow
+pytorch-lightning==1.2.4
```

Comment:
our lightning support is based off of `1.2.0` can we have the same version here? should help with the API when developing harness.

Reply:
1.2.1 through 1.2.4 are just "standard weekly patch releases" and mostly include bug fixes (https://github.com/PyTorchLightning/pytorch-lightning/releases). Happy to use `1.2.0` for now but IMO hardcoding a single specific minor version of PTL to support is probably not going to be tenable.